src/reanimated2/mutables.ts

```diff
@@ -85,10 +80,14 @@ export function makeMutable<Value>(initial: Value): Mutable<Value> {
       }
     },
     get value(): Value {
-      if (!SHOULD_BE_USE_WEB) {
-        return uiValueGetter(mutable);
+      if (SHOULD_BE_USE_WEB) {
+        return value;
       }
-      return value;
+      const uiValueGetter = executeOnUIRuntimeSync(<T>(sv: Mutable<T>): T => {
+        'worklet';
+        return sv.value;
+      });
+      return uiValueGetter(mutable);
```

Comment:
Just out of curiosity, does it work if we inline this function instead of assigning it to a variable? Can we omit the generic types if we do so? Should we rename `sv` to `mutable`?
```suggestion
      return executeOnUIRuntimeSync((mutable) => {
        'worklet';
        return mutable.value;
      })(mutable);
```

Reply:
Also, do we want to automatically workletize the function passed to `executeOnUIRuntimeSync` as we already do for `runOnUI`? Obviously, this can be added in a separate PR.

Reply:
1. Absolutely, we can inline this function instead of assigning it to a variable. However, personally, I find it more readable this way 🥹. But if you prefer, we can certainly change it. We can inline it only to this form:
```js
return executeOnUIRuntimeSync((sv: Mutable<Value>) => {
    'worklet';
    return sv.value;
})(mutable);
```
2. Yes, we can remove the generic type. I forgot about it, good catch! 👍
3. If we change the name to `mutable`, there will be a conflict with line 72
<img width="733" alt="image" src="https://github.com/software-mansion/react-native-reanimated/assets/36106620/c86cc02c-07ae-4e3d-b96b-ce1c30ac36ca">
4. auto-workletization - done ✅

Reply:
1. I like the current version ✅ 
2. Thanks!
3. That's right, thanks for explaining!
4. Thanks!